### ISSUE TITLE
rest-service should ignore itself

### ISSUE THREAD

**Issue by TheCodeDestroyer** [Gillespie59/eslint-plugin-angular#465]
If I write my own service and try to enforce that one throughout other services,
it fails on itself.

**Comment by Gillespie59** [comment]
Can you send some code, in order to explain in detail your issue ?

**Comment by TheCodeDestroyer** [comment]
As an example if i write my custom $http wrapper called `cmnHttpSvc` and I use it in `.eslintrc` like this:
```json
"angular/rest-service": [ 2, "cmnHttpSvc" ],
```
I get errors from that service that it's using `$http`.
The service looks like this right now:

```js
angular.module('module.services')// eslint-disable-line angular/rest-service
.service('cmnHttpSvc', function($http) {
```
